src/core/structures/acroform/PDFRadioButton.ts

```diff
@@ -0,0 +1,78 @@
+import { PDFAnnotation, PDFArray, PDFDict, PDFName } from 'src/core';
+import { PDFAcroButton } from './index';
+
+class PDFRadioButton extends PDFAcroButton {
+  static fromDict(dict: PDFDict): PDFRadioButton {
+    return new PDFRadioButton(dict);
+  }
+
+  readonly dict!: PDFDict;
+
+  protected constructor(dict: PDFDict) {
+    super(dict);
+  }
+
+  Opt(): PDFArray | undefined {
+    return this.dict.lookupMaybe(PDFName.Opt, PDFArray);
+  }
+
+  getOptions(): PDFName[] {
+    const optionsArray = this.Opt();
+    return optionsArray ? optionsArray.lookupElements(PDFName) : [];
+  }
+
+  NoToggleToOff(): boolean {
+    return !!(this.getFlags() & (1 << 14));
+  }
+
+  RadiosInUnison(): boolean {
+    return !!(this.getFlags() & (1 << 25));
+  }
+
+  Kids(): PDFArray {
+    return this.dict.lookup(PDFName.Kids, PDFArray);
+  }
+
+  getKids(): PDFDict[] {
+    return this.Kids().lookupElements(PDFDict);
+  }
+
+  getValue(): PDFName {
+    const value = this.dict.lookup(PDFName.V);
+    if (!value) {
+      return PDFName.Off;
+    }
+    return this.dict.lookup(PDFName.V, PDFName);
+  }
+
+  _getChildOnAppearanceState(index: number): PDFName {
```

Comment:
Since we're using TypeScript, if these fields are supposed to be private, can we mark them using the `private` keyword instead of underscore prefixes?

Reply:
Updated